File: src/main/java/info/opensigma/module/impl/ClickGuiModule.java
package info.opensigma.module.impl;

import info.opensigma.module.Module;
import info.opensigma.ui.clickgui.ClickGui;
import org.lwjgl.glfw.GLFW;

public class ClickGuiModule extends Module {

    private final ClickGui clickGui = new ClickGui();

    public ClickGuiModule() {
        super("ClickGui", "ClickGui", GLFW.GLFW_KEY_RIGHT_SHIFT);
    }

    @Override
    public void onEnable() {
        client.openScreen(clickGui);

        toggle();
    }

}

File: src/main/kotlin/info/opensigma/module/impl/ClickGuiModule.kt
package info.opensigma.module.impl

import info.opensigma.module.Module
import info.opensigma.ui.clickgui.ClickGui
import org.lwjgl.glfw.GLFW

class ClickGuiModule : Module("ClickGui", "ClickGui", GLFW.GLFW_KEY_RIGHT_SHIFT) {

    private val clickGui = ClickGui()

    override fun onEnable() {
        client?.openScreen(clickGui)

        toggle()
    }

}